backend/src/org/akvo/dash/import/csv.clj

```diff
@@ -32,7 +32,7 @@
   (format "CREATE %s %s (%s, %s)"
           (if temp? "TEMP TABLE" "TABLE")
           t-name
-          (if temp? "rnum serial" "rnum integer")
+          (if temp? "rnum serial primary key" "rnum integer primary key")
```

Comment:
Serial is just a short-hand for having an `integer` column and a default value a call to a _sequence_ function. In the case of the non-temp table, we're not going to add more rows, so we no need to generate more `rnum` values via call to the sequence function. It's a minor optimization to not have more dependent objects than needed.

https://www.postgresql.org/docs/8.1/static/datatype.html#DATATYPE-SERIAL